analyses/molecular-subtyping-embryonal/README.md

```diff
@@ -27,14 +27,25 @@ Files will be regenerated using the symlinked files in `data` by default when ru
 bash run-embryonal-subtyping.sh
 ```
 
+`00-embryonal-select-pathology-dx.Rmd` is not run via this module's shell script, as it should be run locally, tied to `release-v17-20200908`, and should not be re-rendered when there are changes to the underlying `pbta-histologies.tsv` file in future releases (see [Folder content](#folder-content) and [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)).
```

Comment:
Not easily commented on in this PR, but you should update line 5 to reflect the release that you used here.